### PR TITLE
Add citation info to README

### PR DESCRIPTION
**Pull request by krishnakatyal**
Added information about citation.

## PR Summary
Added information about citation 
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by krishnakatyal** [comment]
@QuLogic kindly review this PR

File: README.rst [hunk 1]
```diff
@@ -92,3 +92,9 @@ You want to tell us about it – best of all!
 Start at the `contributing guide <http://matplotlib.org/devdocs/devel/contributing.html>`_!
 
 Developer notes are now at `Developer Discussions <https://github.com/orgs/matplotlib/teams/developers/discussions>`_ (Note: For technical reasons, this is currently only accessible for matplotlib developers.)
+
+Citing Matplotlib
+==========
+Matplotlib is the brainchild of John Hunter (1968-2012), who, along with its many contributors, have put an immeasurable amount of time and effort into producing a piece of software utilized by thousands of scientists worldwide.
+If Matplotlib contributes to a project that leads to a scientific publication, please acknowledge this work by citing the project.
+`A ready-made citation entry <https://matplotlib.org/citing.html/>`_ is available.
```
Comment (dstansby) [inline README.rst]: There's an extra backslash at the end of the link which means it's broken currently.

**Comment by krishnakatyal** [comment]
@dstansby thank you
i am again sending a PR by correcting the error

**Comment by krishnakatyal** [comment]
@dstansby Done. 
Corrected the broken link.

File: README.rst [hunk 2]
```diff
@@ -92,3 +92,9 @@ You want to tell us about it – best of all!
 Start at the `contributing guide <http://matplotlib.org/devdocs/devel/contributing.html>`_!
 
 Developer notes are now at `Developer Discussions <https://github.com/orgs/matplotlib/teams/developers/discussions>`_ (Note: For technical reasons, this is currently only accessible for matplotlib developers.)
+
+Citing Matplotlib
+==========
```
Comment (bingyao) [inline README.rst]: I think this underline (equal signs) would be better as:
```rst
Citing Matplotlib
=================
```

Comment (krishnakatyal) [reply README.rst]: @bingyao done

**Comment by krishnakatyal** [comment]
@timhoffm will this PR get merge?

**Comment by timhoffm** [comment]
@krishnakatyal I'm not the right person to decide this as I don't have much to do with citations. This should be decided by other core devs.

**Comment by krishnakatyal** [comment]
@timhoffm okay!
can you mention other core devs to review the PR

**Comment by story645** [comment]
I agree with @timhoffm that especially for the readme, a citation heading & a link to or embedding of the citation page is enough. Some of the it's important to cite language might be better added to the citation page in the docs.

**Comment by tacaswell** [comment]
I agree with @story645 and @timhoffm that this is a bit wordy, but the threshold for docs changes is "does it make it better" which this definitely does so am :+1: on this going in as-is.

I am wary of merging this before we sort out why circle is faliing (which looks like problems on their side for now).

**Comment by krishnakatyal** [comment]
@tacaswell  hi
could you merge this PR?
making it less worthy will not be a good idea as it will not make the doc as better as it is making now

**Comment by ImportanceOfBeingErnest** [comment]
I think the problem that is mentionned by people as "too wordy" is not the number of words, but how that sentence comes across. Matplotlib is above all a technical tool that is used and cited by scientists. A paragraph telling where information on citations are to be found is not the right place to state how great, useful and wide-spread this tool is - because none of this is a reason to cite something. Citations are about hard facts. But words like "immeasurable" and "thousands" imply the contrary.  


I would remove the complete first sentence and let the second read

    If Matplotlib contributes to a project that leads to a scientific publication,
    consider citing Matplotlib.

Then continue with the link.

**Comment by dstansby** [comment]
Thanks for the improvement @krishnakatyal - I'm going to merge this, and then make the suggested changes in a new PR.
